src/Categories/Functor/Limits.agda

```diff
@@ -0,0 +1,82 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Functor.Limits where
+
+open import Level
+
+open import Categories.Category
+open import Categories.Functor
+open import Categories.Object.Terminal
+open import Categories.Object.Initial
+
+open import Categories.Category.Construction.Cones
+open import Categories.Category.Construction.Cocones
+
+private
+  variable
+    o ℓ e : Level
+    𝒞 𝒟 ℐ : Category o ℓ e
+
+-- Whiskering of Functors onto Cones
+_∘Cone_ : (F : Functor 𝒞 𝒟) → {J : Functor ℐ 𝒞} → Cone J → Cone (F ∘F J)
```

Comment:
sorry to not finish reviewing in one shot. I think there are three potential improvements to make for the composition:

1. this composition is actually implemented here: 
https://github.com/agda/agda-categories/blob/1e02a1c6254f5d950c46379cf879af396d1c795c/src/Categories/Adjoint/RAPL.agda#L47-L54
so it might just be refactored here.
2. there is another file: https://github.com/agda/agda-categories/blob/master/src/Categories/Functor/Construction/Limit.agda
should we have two of these files or merge them into one?
3. the composition for cocone can just be obtained from duality. `lapc` is obtained via `rapl` by
https://github.com/agda/agda-categories/blob/1e02a1c6254f5d950c46379cf879af396d1c795c/src/Categories/Adjoint/Properties.agda#L175-L176
consider replacing cocone composition with this duality style.